src/preact/context.js

```diff
@@ -77,3 +85,35 @@ export function useAmpContext() {
   const AmpContext = getAmpContext();
   return useContext(AmpContext);
 }
+
+/**
+ * Whether the calling component should currently be in the loaded state.
+ *
+ * @param {!Loading|string} loadingProp
+ * @return {boolean}
+ */
+export function useLoad(loadingProp) {
+  const loadingRef = useRef(false);
+
+  const {loading: loadingContext, renderable} = useAmpContext();
+
+  const loading = loadingReducer(loadingProp, loadingContext);
+
+  // Compute whether the element should be loading at this time.
+  const load =
+    // Explicit instruction to unload.
+    loading == Loading.UNLOAD
+      ? false
+      : // Explicit instrunction to load.
```

Comment:
nit: instrunction --> instruction

Reply:
this is getting embarrassing.

Reply:
lol, if IDEs had spellcheck then I'd never have comments on your code :)